osquery/CMakeLists.txt

```diff
@@ -69,6 +69,10 @@ else()
   ADD_OSQUERY_LINK_CORE("glog")
 endif()
 
+ADD_OSQUERY_LINK_CORE("broker")
```

Comment:
These lines should do most of the work of the `FindX` CMake libraries. We should remove those files in `cmake/*`.